client/stylesheets/shared/_global.scss

```diff
@@ -53,4 +53,11 @@ body.woocommerce-page {
 			color: $studio-white;
 		}
 	}
+
+	.woocommerce-ellipsis-menu__content {
```

Comment:
I think we could move this styling directly to `packages/components/src/ellipsis-menu/style.scss`.  Since we already have precedent for setting styles of child GB components in there and we would presumably want all consumers of the ellipsis menu to use our same styling here.

Reply:
Agreed. In my youth (this morning) I didn't understand the packages well enough to recognize this was the move, but after my adventures with `SelectControl` labels I am older and wiser.